app/components/trainees/confirmation/programme_details/view.rb

```diff
@@ -30,9 +30,9 @@ def age_range
         end
 
         def programme_type
-          return @not_provided_copy if trainee.record_type.blank?
+          return @not_provided_copy if trainee.training_route.blank?
 
-          trainee.record_type.humanize
+          trainee.training_route.humanize
```

Comment:
![image](https://user-images.githubusercontent.com/470137/109638964-86893c00-7b46-11eb-9644-b221b0d4fe9d.png)


```suggestion
t("activerecord.attributes.trainee.training_routes.#{trainee.training_route}")
```

missing `-`